src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
> test the package in current development, e.g. when in your working directory you're trying with some beta version, not yet committed

I've been on the fence about simply referencing the project and not using the nuget in the demo projects as its much easier to maintain. I really want to have the build server version the package automatically which might make having a nuget reference a little painful